agent/grpc-external/services/peerstream/replication.go

```diff
@@ -36,10 +36,14 @@ import (
 // If there are no instances in the event, we consider that to be a de-registration.
 func makeServiceResponse(
 	logger hclog.Logger,
+	mst *MutableStatus,
 	update cache.UpdateEvent,
 ) (*pbpeerstream.ReplicationMessage_Response, error) {
+	serviceName := strings.TrimPrefix(update.CorrelationID, subExportedService)
+	sn := structs.ServiceNameFromString(serviceName)
 	csn, ok := update.Result.(*pbservice.IndexedCheckServiceNodes)
 	if !ok {
+		logger.Error("did not increment or decrement exported services count", "service_name", serviceName)
```

Comment:
this log is probably redundant, given that error handling should indicate in the logs that a service was not processed